Traceback (most recent call last):
  File "<string>", line 5, in <module>
  File "/home/sc.uni-leipzig.de/ci72buri/projects/vae/Leipzig_big_data_praktikum_gene_expression_autoencoder/./train_vae.py", line 412, in <module>
    t
NameError: name 't' is not defined
